data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,129 @@ mission "Coalition: Outskirts Gauger Ready"
 			`	While the Arachi work on the outfit, she explains that they used the probe's algorithms to have the scanners excel at finding basic energy signatures, such as heat or energy levels of other ships. "Made to check for complex machinery like outfits, or recognize specific cargo types, it was not, but still find uses for its current form, I'm sure you will."`
 			`	When everything's ready, you two head back inside your ship, and the elevator brings you back up. You fly back to the main landing area, and she pulls close to you just before you open your hatch. "Now, still... experimental, the outfit is, and provide the Heliarchs with a potentially faulty product, we do not wish. So, naturally, not yet registered it with them, we have. Provide you with more copies, we cannot, so careful not to lose this one, you should be." She thanks you again for helping out, and leaves for her workplace.`
 				decline
+
+mission "Sativa: Struggling Author 1"
+	minor
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		random < 30
+		credits > 140
+		has "Coalition: Contributor: done"
+	on offer
+		conversation
+			`	Upon entering the spaceport, you walk past a bookstore, currently holding a book signing event for an author known as 'Ann Oryza Sativa'. Nobody pays attention to her. It appears as if she is a struggling author.`
+			choice
+				`	(Ignore her.)`
+					decline
+				`	(Buy one of her books.)`
+					goto "buy her book"
+				`	(Sit down and talk to her.)`
+			label "talk to her"
+			`	You sit down on the table, next to Ann. She greets you with a smile. "Hello! Would you like to buy one of my books?" she asks.`
+			choice
+				`	"What are your books about?"`
+					goto "what are her books about"
+				`	"No, I'm not interested. How has life been as an author though?"`
+					goto "life as an author sucks"
+			label "what are her books about"
+			`	"I mostly write science fiction books. Today, I'm here to promote 'Beyond the Hyperdrive' a story in which a group of humans steal a bunch of jump drives from the Quarg. If I continue past this point I'd be spoiling it. Read it for yourself, you'll like it!"`
+			choice
+				`	"I'm interested. Wait here, I'll go buy myself a copy."`
+					goto "buy her book"
+				`	"Cool. How has life been lately as an author?"`
+			label "life as an author sucks"
+			`	"Bad," says Ann. "I've never had a bestseller or even a book that surpasses 10,000 copies sold."`
+			choice
+				`	(Tell her about the Coalition.)`
```

Comment:
This is a bit too quick. Could use a bit more of scene setting and talking with her to reach a point where the player character would think she's sounding similar to the Coalition. Maybe you can inquire if she doesn't like the Quarg, and she can give you a similar rundown the Heliarchs give, that the Quarg are limiting people, or maybe that she's just scared of them due to whatever.

Another, longer way to do this (reaching the Coalition), would be for her to ask the player if they could try and promote her books on alien markets. You could then have a similar setup as with the Ringworld Debris mission, where upon first landing on a Hai/Korath/Wanderer/Gegno (maybe a Quarg one too for the heck of it, have them just sigh when seeing it paints them negatively or something) world you can try and see if they like it. Then on a Coalition world the Kimek pick it up and it really gets going.

Reply:
If I were to redo the chain so that we'd be taking Ann to Hai, Korath, Wanderer, Gegno, Quarg, and then Kimek space, where would I put the mission text? In `coalition missions.txt` or in a different file seeing as it now encompasses a lot more alien species than just the Kimek now.

Reply:
I figured in that case you'd just be bringing about copies of her book, but regardless of whether it's that or bringing her along, since the bulk would still be taking place in Coalition space then it'd be there still I figure. The only other place would be for `human missions.txt` since it starts in human space, but given the endpoint is Coalition I think that's more appropriate.
The interactions with the other species would be brief, and more there as just to cover for the cases where you bring her/her work to places other than the Coalition.

Reply:
>I don't get what this means. Ann isn't aware of the existence of the Coalition, probably only knows about the Quarg, and maybe after Hai Reveal, the.. Hai.

As in, write a bit more dialog that'd lead the player to think of telling her about the Coalition as an option. "Sound similar to the Coalition" as in maybe say something in line with what the Coalition says. She summarizes that the story has some theft of Quarg JDs going on, so that could be something you can inquire her on, and maybe learn she's not very fond of the Quarg. I meant that as a possible way of determining she'd be more in line with the Coalition, as they have a history of being against the Quarg.

Reply:
> > I don't get what this means. Ann isn't aware of the existence of the Coalition, probably only knows about the Quarg, and maybe after Hai Reveal, the.. Hai.
> 
> As in, write a bit more dialog that'd lead the player to think of telling her about the Coalition as an option. "Sound similar to the Coalition" as in maybe say something in line with what the Coalition says. She summarizes that the story has some theft of Quarg JDs going on, so that could be something you can inquire her on, and maybe learn she's not very fond of the Quarg. I meant that as a possible way of determining she'd be more in line with the Coalition, as they have a history of being against the Quarg.

Ah, cool. So I'll make it so that Ann hates the Quarg. Then First Last tells Ann "you know, there's a group of aliens who don't like the Quarg. They're called the Coalition."

And then Ann will hopefully be interested in coming over for a visit. Then FL can tell the Heliarchs that "she doesn't like the oppressors (the Quarg) just like you".